BEP84.md

```diff
@@ -0,0 +1,216 @@
+
+## 2. Abstract
+Currently, if a user wants to issue and bind a BEP2 token with an existing BEP20 token, it has to do a set of complex operations, including issue BEP2 tokens, sending a bind transaction and approving the binding request. The BEP will bring a mechanism to simplify the above process. In the new mechanism, what anyone can do this is just sending a transaction to Binance Smart Chain. 
+
+## 3. Motivation
+Current bind mechanism is based on the context that our community members are on the Binance Chain and they want to extend their tokens to the Binance Smart Chain. However, with the evolution of our community, things changed. In most cases, users issue bep20 on the Binance Smart Chain first without considering whether they will issue BEP2 on the Binance Chain or not,  so a new mechanism to conveniently extend BEP20 assets to the Binance Chain is required. In addition, the new mechanism will encourage users to extend their assets to the Binance Chain which is very helpful to flourish the Binance Chain community. 
+
+## 4. Status
+This BEP is under draft.
+ 
+## 5. Specification
+
+Two new permissionless methods will be imported into TokenManager contract:
+Mirror: If a BEP20 contract is not bound to any BEP2 token, anyone can call the mirror method to automatically issue a BEP2 token and bind them. The user is not required to have any BEP20 token and just needs to pay enough BEP2 issue fee. Besides, the BEP20 contract is even not required to implement getOwner method. After binding, all circulation is on BSC.
+Sync: For a BEP20 token which has been mirrored to BC, anyone can call sync method to balance the total supply on BC and BSC. For example, someone mint some BEP20 token, after calling sync method,  the equivalent token will be minted on BC and transferred to the pure-code-controlled-escrow address. If someone burn some BEP20 token, after calling sync method, the equivalent token will be burned on BC from the pure-code-controlled-escrow address
+
+### 5.1 TokenManager
+
+#### 5.1.1 Mirror
+
+##### 5.1.1.1 Parameters
+
+| **Param Name**    | **Type** | **Description**   |
+| ------------ | -------- | ---------------------- |
+| BEP20Addr    | Address  | BEP20 contract address |
+| ExpiredTime  | uint64   | The deadline to deliver this package on BC |
+| msg.value    | uint256  | Sum of cross chain fee and mirror fee |
+
+##### 5.1.1.2 Pre-check
+
+1. Ensure the BEP20 token hasn’t been bound before and is not in mirror pending status.
+2. Ensure the BEP20 symbol follows the BEP2 symbol requirements.
+3. Ensure the equivalent total supply is no greater than Max BEP2 total supply.
+4. Ensure msg.value >= MirrorFee + CrossChainFee and msg.value = N * 10^10
+5. Ensure expired time is earlier than one day later and no earlier than two minutes later.
+
+##### 5.1.1.3 Core Mechanism
+
+1. Transfer relayFee to TokenHub
+2. Mark the BEP20 token is in mirror pending status
```

Comment:
`Mark the BEP20 token is in mirror pending status` ==> `Mark the BEP20 token as mirror pending status`